README.md

```diff
@@ -1,10 +1,18 @@
 # decorate-spec
 
-A GitHub action for generating a "decorated" openapi spec for readme.com. Will take your openapi spec and stainless config and generate a file in the current working directory called `CUSTOMER-openapi.documented.json`, where `CUSTOMER` is your name.
+A GitHub action for generating a "decorated" openapi spec for readme.com. Will take your openapi spec and generate a file in the current working directory called `CUSTOMER-openapi.documented.json`, where `CUSTOMER` is your name.
 
-## Example Usage
+## Setup
 
-Here is an example GitHub action workflow that will use this action to generate the decorated openapi spec and then upload it to readme.com. This file should be stored in `.github/workflows/`
+1. Copy the example from below into a GitHub workflow file (e.g. `.github/workflows/decorate.yml`)
+2. Replace `CUSTOMER` with your company's name
+3. Replace `PATH_TO_SPEC` with the relative path to your openapi spec. This path should be relative to the root of the repo.
+4. Add [GitHub actions secrets storing your credentials](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+   - `secrets.STAINLESS_CONTAINER_TOKEN`: You should've been given a GitHub token, this should be called that.
+   - `secrets.README_TOKEN`: Your API token for README.com.
+   - `secrets.README_DEFINITION_ID`: According to readme's documentation can be obtained by "clicking edit on the API definition on your project API definitions page".
```

Comment:
```suggestion
   - `secrets.README_DEFINITION_ID`: According to Readme's documentation, this can be obtained by "clicking edit on the API definition on your project API definitions page".
```

Reply:
Maybe also say a word how this will be used?